frontend/lib/norent/letter-builder/rent-periods.tsx

```diff
@@ -23,12 +25,21 @@ function getCurrentRentNonpaymentPeriods(s: AllSessionInfo): string[] {
   );
 }
 
-function getRentNonpaymentChoices(
+// LA residents' protections extend beyond those in other cities.
+function addCaveatForLA(paymentDate: GraphQLDate) {
+  let caveat = "";
+  if (new Date(paymentDate) >= END_DATE_OF_CA_STATE_PROTECTIONS) {
+    caveat = " " + li18n._(t` (only for City of Los Angeles residents)`);
```

Comment:
hmmm @samaratrilling looks like this would include two consecutive spaces in the `caveat` string. Assuming that is not intentional?

Reply:
Ah, actually, the one inside the `t` won't count, but I removed it anyway for clarity.